docs/quick-start/ignite-commands.md

```diff
@@ -66,7 +66,7 @@ ignite new {Project name}
 
 Generate a new React Native project with Ignite.
 
-`ignite n` uses `react-native init`, then adds files specific to Ignite.
+`ignite new` uses `react-native init`, then adds files specific to Ignite.
```

Comment:
technically you can use `ignite n` also, but what you have is better for concept.